spec/facility_spec.rb

```diff
@@ -23,4 +29,173 @@
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+
+  describe '#register' do
+    it 'can register a vehicle' do
+      cruz = Vehicle.new({vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice} )
+
+      @facility.add_service('Vehicle Registration')
+      @facility.register_vehicle(cruz)
+
+      expect(@facility.vehicle_registered?(cruz)).to be(true)
+    end
+
+    it 'will not register a vehicle if the facility does not offer new vehicle registration' do
+      cruz = Vehicle.new({vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice} )
+
+      @facility.register_vehicle(cruz)
+
+      expect(@facility.vehicle_registered?(cruz)).to eq(false)
+    end
+
+    it 'will store a new plate type once the vehicle is registered' do
+      cruz = Vehicle.new({vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice} )
+      @facility.add_service('Vehicle Registration')
+      @facility.register_vehicle(cruz)
+
+      expect(cruz.plate_type).to eq(:regular)
+    end
+
+    it 'will store the date the vehicle was registerd on' do
+      cruz = Vehicle.new({vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice} )
+      @facility.add_service('Vehicle Registration')
+      @facility.register_vehicle(cruz)
+
+      expect(cruz.registration_date).to eq(Date.today)
+    end
+  end
+
+
+  describe '#add_service' do
+    it 'can add a service' do
+      @facility.add_service('Written Test')
+
+      expect(@facility.services).to eq(['Written Test'])
+    end
+  end
+
+  describe '#cost' do
```

Comment:
I'm glad you still tested the fees here, but again, might make more sense to put them in the register_vehicle describe block.